packages/preferences/src/browser/preferences-contribution.ts

```diff
@@ -61,6 +63,12 @@ export class PreferencesContribution extends AbstractViewContribution<Preference
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
+        if (isOSX && !isFirefox) {
+            keybindings.registerKeybinding({
+                command: CommonCommands.OPEN_PREFERENCES.id,
+                keybinding: 'cmd+,'
+            });
+        }
```

Comment:
Should not be `else` here?

Reply:
This is intended: E.g. on Mac OS, in Chrome or Electron, we'll register both `cmd+,` and `ctrl+,`. The first one is the new default keybinding, while the second one is there for people who bot used to typing `ctrl+,` on Mac OS.

Side note: In the `File` > `Settings` > `Open Preferences` menu entry, only the first keybinding shows up. (I double-checked that both keybindings work as expected, but that only the first is shown in the menu.)

Reply:
ok

Reply:
FYI: this change will disable the default Chrome binding for opening Chrome preferences on macOS. Was it on purpose?

Reply:
Yes, in order to follow VS Code's convention, we'd like to replace the behavior of `cmd+,` in browsers. (This only works in Chrome though -- Firefox will continue to open browser preferences, which is why we set a different default keybinding there).

Would you be surprised that hitting `cmd+,` in Theia in Chrome now opens the Theia preferences and no longer the Chrome preferences?

Reply:
I thought we follow VS Code's convention in electron only and not disable/override the browser bindings.

Reply:
Good point. My understanding was that we try to follow VS Code's convention everywhere, but that in some browsers, it's sometimes impossible to override browser bindings (e.g. it's impossible to override `cmd+,` in Firefox) in which case we fall back on alternative bindings, but I may be wrong.

Reply:
> we try to follow VS Code's convention everywhere

I thought, no. My understanding was the following; we mimic the VS Code keymap in electron and leave the rest as is. For instance, why would we ever distinguish between macOS and other platforms when using Gitpod in Chrome.

CC: @svenefftinge

Reply:
Keybindings are part of a user's muscle memory. So we want to meet expectations as much as possible. I.e. the only reason to diverge from the default keybindings from VS Code (resp. keymaps like emacs or vim) is that browsers hinder us from handling certain keybindings.
Since CMD+, works in Chrome it should be used.